scripts/ldap/setup_ldap.sh

```diff
@@ -96,8 +95,23 @@ rm -rf /etc/openldap/slapd.d/cn\=config/olcDatabase\=\{2\}mdb.ldif
 if [[ $forceclean == true ]]
 then
   rm -rf /var/lib/ldap/*
+  if rpm -q 'symas-openldap-clients' 
+  then
+    yum remove -y symas-openldap-clients
```

Comment:
These changes should not be merged to production (as openldap comes through RE third party framework) - Can be kept for testing

Reply:
Also - move these changes to init.sh allied ansible 
in provisioner environment, RE will handle the OpenLDAP installation